Core/HLE/sceUtility.cpp

```diff
@@ -690,6 +690,25 @@ static u32 sceUtilityGetSystemParamString(u32 id, u32 destaddr, int destSize)
 	return 0;
 }
 
+static u32 sceUtilityGetSystemParamInt_(u32 id, u32 value)
+{
+	        switch (id) {
```

Comment:
A lot of these lines use spaces for indentation.

I recommend installing editorconfig:
https://marketplace.visualstudio.com/items?itemName=EditorConfigTeam.EditorConfig

And then simply reindent this section.  Tabs for indent, spaces for alignment.

-[Unknown]